ext/memcached_rb/memcached_rb.c

```diff
@@ -1,460 +1,518 @@
-#include <memcached_rb.h>
-#include <memcached_rb_behavior.h>
+#include "memcached_rb.h"
 
-VALUE rb_mMemcached,
-  rb_cServer,
-  rb_eMemcachedError;
+VALUE rb_mMemcached;
 
-ID id_ivar_hostname,
-  id_ivar_port,
-  id_ivar_weight,
-  id_tcp,
-  id_socket;
+static VALUE rb_cServer;
```

Comment:
It's usually good style to leave classes that don't need to be shared between files as `static`.